src/OpenSage.Game/Data/W3d/W3dAnimation.cs

```diff
@@ -17,6 +17,9 @@ internal static W3dAnimation Parse(BinaryReader reader, W3dParseContext context)
             {
                 var result = new W3dAnimation();
 
+                // TODO: generals zero hour assets: UISabotr throw chunktype errors.
```

Comment:
You'll notice `UISabotr_*` files are ignored in `W3dFileTests` for that exact reason. As far as I can tell, they contain corrupt chunks. So I think no need for this comment.